care.js

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 var config = require(__dirname + '/config.js');
 var twitterbot = require(__dirname + '/twitterbot.js');
+var notifier = require('node-notifier');
+var Pomodoro = require(__dirname + '/pomodoro.js');
```

Comment:
@notwaldorf https://github.com/notwaldorf/tiny-care-terminal/pull/73/commits/7c67cd8603b940528287b1c22ed7a9b23cca56e6 What do you think  ?